plugins/copr.py

```diff
@@ -110,6 +102,12 @@ def configure(self):
 
         if config_file is not None:
             copr_plugin_config.read(config_file)
+            if copr_plugin_config.has_option(copr_instance, 'url'):
+                self.copr_url = copr_plugin_config.get(copr_instance, 'url')
+            else:
+                print(_("Warning: No such instance '{}' in configuration file. "
+                        "Using the default one instead '{}'.").format(copr_instance, self.copr_url))
```

Comment:
Use `{0}` and `{1}`. The order of the placeholders does not need to be the same in all translations.